### PR TITLE
add profile

### PR DESCRIPTION
**Pull request by lgimeneza**

### CODE REVIEW

**Comment by MinaZhen** [comment]
mu malamente

File: staff/lgimeneza/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# [Github](https://github.com/lgimeneza)
```
Comment (MinaZhen) [inline staff/lgimeneza/README.md]: ADD # Share

Comment (lgimeneza) [reply staff/lgimeneza/README.md]: Done!!

Comment (albanelia0) [reply staff/lgimeneza/README.md]: Thanks for your comment!!!^^

**Pull request merged**